apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -224,6 +227,25 @@ export class WidgetsController {
     return await this.markMessageAsUsecase.execute(command);
   }
 
+  @ApiOperation({
+    summary: "Mark subscriber feed's all unread messages as read",
+  })
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Post('/messages/feed/markAsRead')
```

Comment:
I'm wondering why can't we modify the `widgets/messages/markAs` endpoint to allow marking all the messages read/seen whatever?
So like if the message ids won't be provided then we will mark all read/seen, otherwise only selected.
@BiswaViraj